contracts/governance/staking/GamifiedTokenStructs.sol

```diff
@@ -38,3 +38,22 @@ struct Quest {
     /// Expiry date in seconds for the quest
     uint32 expiry;
 }
+
+struct SafetyData {
```

Comment:
How come these need to be in here? I am not sure the name makes sense given this was originally for the structs from the parent